build/test.mk

```diff
@@ -60,8 +60,13 @@ test-validate-cli: ## Run cli integration tests
 
 test-functional-all: test-functional-ucp test-functional-kubernetes test-functional-shared test-functional-msgrp test-functional-daprrp test-functional-datastoresrp test-functional-samples ## Runs all functional tests
 
```

Comment:
The second commit in this branch moves the testUtil.go to test/testutil folder. Might be easier to review it separately as it is only renames of package name